definitions/src/instructions.rs

```diff
@@ -28,160 +28,157 @@
+pub const OP_RORIW: InstructionOpcode = 0x79;
+pub const OP_RORW: InstructionOpcode = 0x7a;
+pub const OP_SEXTB: InstructionOpcode = 0x7b;
+pub const OP_SEXTH: InstructionOpcode = 0x7c;
+pub const OP_SH1ADD: InstructionOpcode = 0x7d;
+pub const OP_SH1ADDUW: InstructionOpcode = 0x7e;
+pub const OP_SH2ADD: InstructionOpcode = 0x7f;
+pub const OP_SH2ADDUW: InstructionOpcode = 0x80;
+pub const OP_SH3ADD: InstructionOpcode = 0x81;
+pub const OP_SH3ADDUW: InstructionOpcode = 0x82;
+pub const OP_SLLIUW: InstructionOpcode = 0x83;
+pub const OP_XNOR: InstructionOpcode = 0x84;
+pub const OP_ZEXTH: InstructionOpcode = 0x85;
 // Mop
-pub const OP_WIDE_MUL: InstructionOpcode = 0x76;
-pub const OP_WIDE_MULU: InstructionOpcode = 0x77;
-pub const OP_WIDE_MULSU: InstructionOpcode = 0x78;
-pub const OP_WIDE_DIV: InstructionOpcode = 0x79;
-pub const OP_WIDE_DIVU: InstructionOpcode = 0x7A;
-pub const OP_FAR_JUMP_REL: InstructionOpcode = 0x7B;
-pub const OP_FAR_JUMP_ABS: InstructionOpcode = 0x7C;
-pub const OP_ADC: InstructionOpcode = 0x7D;
-pub const OP_SBB: InstructionOpcode = 0x7E;
-pub const OP_ADCS: InstructionOpcode = 0x7F;
-pub const OP_SBBS: InstructionOpcode = 0x80;
-pub const OP_ADD3A: InstructionOpcode = 0x81;
-pub const OP_ADD3B: InstructionOpcode = 0x82;
-pub const OP_ADD3C: InstructionOpcode = 0x83;
-pub const OP_CUSTOM_LOAD_UIMM: InstructionOpcode = 0x84;
-pub const OP_CUSTOM_LOAD_IMM: InstructionOpcode = 0x85;
-pub const OP_CUSTOM_TRACE_END: InstructionOpcode = 0x86;
+pub const OP_WIDE_MUL: InstructionOpcode = 0x86;
+pub const OP_WIDE_MULU: InstructionOpcode = 0x87;
+pub const OP_WIDE_MULSU: InstructionOpcode = 0x88;
+pub const OP_WIDE_DIV: InstructionOpcode = 0x89;
+pub const OP_WIDE_DIVU: InstructionOpcode = 0x8a;
+pub const OP_FAR_JUMP_REL: InstructionOpcode = 0x8b;
+pub const OP_FAR_JUMP_ABS: InstructionOpcode = 0x8c;
+pub const OP_ADC: InstructionOpcode = 0x8d;
+pub const OP_SBB: InstructionOpcode = 0x8e;
+pub const OP_ADCS: InstructionOpcode = 0x8f;
+pub const OP_SBBS: InstructionOpcode = 0x90;
+pub const OP_ADD3A: InstructionOpcode = 0x91;
+pub const OP_ADD3B: InstructionOpcode = 0x92;
+pub const OP_ADD3C: InstructionOpcode = 0x93;
+pub const OP_CUSTOM_LOAD_UIMM: InstructionOpcode = 0x94;
+pub const OP_CUSTOM_LOAD_IMM: InstructionOpcode = 0x95;
+pub const OP_CUSTOM_TRACE_END: InstructionOpcode = 0x96;
 
-pub const MINIMAL_LEVEL1_OPCODE: InstructionOpcode = OP_UNLOADED;
-pub const MAXIMUM_LEVEL1_OPCODE: InstructionOpcode = OP_CUSTOM_TRACE_END;
-pub const LEVEL2_B_OPCODE: InstructionOpcode = 0xF0;
-pub const MINIMAL_LEVEL2_B_OPCODE2: InstructionOpcode = 0x00;
-pub const MAXIMUM_LEVEL2_B_OPCODE2: InstructionOpcode = 0x00;
+pub const MINIMAL_OPCODE: InstructionOpcode = OP_UNLOADED;
+pub const MAXIMUM_OPCODE: InstructionOpcode = OP_CUSTOM_TRACE_END;
 
-pub const INSTRUCTION_OPCODE_NAMES_LEVEL1: [&str; MAXIMUM_LEVEL1_OPCODE as usize + 1] = [
+pub const INSTRUCTION_OPCODE_NAMES: [&str; MAXIMUM_OPCODE as usize - 15] = [
```

Comment:
Can we use `MINIMAL_OPCODE` instead of 15?

Reply:
Done